src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -6337,6 +6338,100 @@ void AutoCalcDOASControlStrategy(EnergyPlusData &state)
     }
 }
 
+void ReportInfiltrations(EnergyPlusData &state)
+{
+    // SUBROUTINE INFORMATION:
+    //       AUTHOR         Yueyue Zhou
+    //       DATE WRITTEN   July 2021
+
+    // PURPOSE OF THIS SUBROUTINE:
+    // This subroutine currently creates the values for standard Infiltration object level reporting
+
+    // METHODOLOGY EMPLOYED:
+
+    // REFERENCES:
+
+    using DataHVACGlobals::CycleOn;
+    using DataHVACGlobals::CycleOnZoneFansOnly;
+    int j;  // Loop Counter
+    int NZ; // A pointer
+    // SUBROUTINE PARAMETER DEFINITIONS:
+    static std::string const RoutineName("ReportInfiltration");
+
+    // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+    Real64 AirDensity; // Density of air (kg/m^3)
+    Real64 CpAir;      // Heat capacity of air (J/kg-C)
+    Real64 TotalLoad;  // Total loss or gain
+    Real64 H2OHtOfVap; // Heat of vaporization of air
+    auto &Zone(state.dataHeatBal->Zone);
+    auto &Infiltration(state.dataHeatBal->Infiltration);
+    auto &TimeStepZoneSec(state.dataGlobal->TimeStepZoneSec);
+    auto &TimeStepZone(state.dataGlobal->TimeStepZone);
+    for (j = 1; j <= state.dataHeatBal->TotInfiltration; ++j) {
+
+        NZ = state.dataHeatBal->Infiltration(j).ZonePtr;
+        if (state.dataAirflowNetwork->SimulateAirflowNetwork == AirflowNetwork::AirflowNetworkControlSimpleADS) {
+            // CR7608 IF (TurnFansOn .AND. AirflowNetworkZoneFlag(NZ)) ADSCorrectionFactor=0
+            if ((state.dataZoneEquip->ZoneEquipAvail(NZ) == CycleOn || state.dataZoneEquip->ZoneEquipAvail(NZ) == CycleOnZoneFansOnly) &&
+                state.dataAirflowNetwork->AirflowNetworkZoneFlag(NZ))
+                continue;
```

Comment:
Yes, I expect it to leave everything 0. It is done in this way because I intended to follow the zone reporting variable equations where a factor of 0 was applied to the calculations. Skipping the remaining of function looks more straightforward to me and should have the same effect.